noxfile.py

```diff
@@ -47,7 +47,7 @@ def tests_packaging(session: nox.Session) -> None:
     """
 
     session.install("-r", "tests/requirements.txt", "--prefer-binary")
-    session.run("pytest", "tests/extra_python_package")
+    session.run("pytest", "tests/extra_python_package", *session.posargs)
```

Comment:
This is completely unecessary. I just found it useful to run `nox -s tests_packaging -- -vvv` for debugging so I left the change behind in case you also find it useful.

Reply:
Sounds good, these should be passed through.